docs/obs/dm.rst

```diff
@@ -0,0 +1,116 @@
+.. include:: ../references.txt
+
+.. _obs_dm:
+
+Data Management
+===============
+
+Classes
+-------
+
+Gammapy helps with data management using a multi-layered set of classes:
+
+* The `~gammapy.obs.DataManager` represents a configuration (usually read
+  from a YAML file) of directories and index files specifying where
+  data is available locally and remotely and in which formats.
+* The `~gammapy.obs.DataStore` represents data files in a given directory
+  and usually consists of two things: a `~astropy.table.Table`
+  that contains the location, content, size, checksum of all files
+  and a `~gammapy.obs.ObservationTable` that contains relevant parameters
+  for each observation (e.g. time, pointing position, ...)
+* The actual data and IRFs are represented by classes,
+  e.g. `~gammapy.data.EventList` or `~gammapy.irf.EffectiveAreaTable2D`, ...
+
+The job of the DataManager and DataStore is to make it easy and fast
+to locate files and select subsets of observations.
+
+.. code-block:: python
+
+    >>> from gammapy.obs import DataManager
+    >>> data_manager = DataManager.from_yaml('data-register.yaml')
+    >>> data_store = data_manager['hess-hap-prod01']
+    >>> data_store.filename(obs_id=89565, filetype='AEFF')
+    /Users/deil/work/_Data/hess/fits/hap-hd/fits_prod01/std_zeta_fullEnclosure/run089400-089599/run089565/hess_aeff_2d_089565.fits.gz
+
+
+In addition, these classes contain convenience properties and methods that
+actually load the data and IRFs and return objects of the appropriate class
+
+.. code-block:: python
+
+    >>> aeff = data_store.load(obs_id=89565, filetype='AEFF')
+    >>> aeff.__class__.name
+    TODO: gammapy.irf.EffectiveAreaTable2D
+
+
+Command line tools
+------------------
+
+* ``gammapy-data-manage`` -- Manage data locally and on servers
```

Comment:
this points to 'gammapy-data-manage = gammapy.obs.data_manager:main', which does not exist